src/test/codegen/sanitizer-no-sanitize-inlining.rs

```diff
@@ -13,7 +13,7 @@
 #![feature(no_sanitize)]
 
 // ASAN-LABEL: define void @test
-// ASAN:         tail call fastcc void @random_inline
+// ASAN:         tail call fastcc
```

Comment:
The name is relevant insofar it prevents unintentionally matching on a
different call, especially that `-Zsanitizer=address` does insert calls to
`@__asan_init` and `@__asan_version_mismatch_check_v8`.

As mentioned earlier, the issue is that check unnecessarily looks for fastcc
which is no longer there. I would relax the pattern and retain the no_mangle:

```
// ASAN:         call {{.*}} @random_inline
```

Reply:
Good point - thanks a lot